assignment/bundles/resources/assignment.properties

```diff
@@ -1112,3 +1112,15 @@ subm.conf=Submission Confirmation
 subm.notif=Submission Notification
 submissionConfirmAlert=If you are ready to submit the completed assignment, click "Submit". Otherwise, click "Edit" and you will then be able to make any changes or save it as a draft.
 submissionConfirmMessage=You are about to submit this assignment for grading. Please check that the following items have been completed before submitting.
+
+email.reminder.hello=Hello {0}
+email.reminder.subject=Assignment ''{0}'' Due on {1}
+email.reminder.duewithin=Reminder: An assignment of yours is due within {0}
+email.reminder.assignment=Assignment : {0}
+email.reminder.duedate=Due        : {0}
+email.reminder.class=Class      : {0}
+email.reminder.niceday=Have a nice day!
+email.reminder.andhours=and {0} hours.
+email.reminder.hours={0} hours.
+email.reminder.days={0} days
+email.reminder.dateformat=MMM dd, yyyy @ hh:mm aa
```

Comment:
The date format here is slightly different than the original, will this be a problem?

`MMM dd, yyyy @ hh:mm aa` vs `MMM dd, yyyy '@' hh:mm aa`

Reply:
In my tests it doesn't seem to make a difference, but I just added the single quotes back in to minimize changes.

Reply:
Okay, thanks @curtisvo!